### DISCUSSION TITLE
[Unstable] [Feature Request] Add CheckTalentAction to the event editor

### DISCUSSION THREAD

**Discussion by Udrakan** [FakeFishGames/Barotrauma#9841] [Ideas and suggestions]
**Description**
Having this would give us a chance to mod in good RP opportunities in events tied to talents. Please, consider adding it to the event editor.
![image](https://user-images.githubusercontent.com/59519434/188945028-0b7a22ac-d9b2-404a-8ac7-04f8e781bdac.png)

**Comment by ahzidaljun** [comment]
This sounds like it'd be big for modding, especially with how descriptive the talent names can be for how a character is developing over the course of a campaign.
